src/app.js

```diff
@@ -1,9 +1,32 @@
 angular.module('smartvid', [
   'ngRoute',
   'ngCookies',
+  'ngSanitize',
+  'jm.i18next',
   'templates'
-]).run(function ($cookies, utils) {
+]).run(function ($cookies, utils, $rootScope) {
   if (!$cookies.guid) {
     $cookies.guid = utils.createGuid()
   }
+
+  //TODO - Temp have this in as a stub.
+  $rootScope.user = {
+    firstName: 'Joe',
+    lastName: 'Dirt',
+    role: 'Super',
+    account: {
+      name: "Acme Construction"
+    }
+  };
 })
+
+angular.module('smartvid').config(['$i18nextProvider', function($i18nextProvider) {
```

Comment:
There will be more configs to come. What do you think about a config folder with files like `i18next.js` and `filepicker.js` or whatever config files we need?